### PR TITLE
docs(readme): Add gimbal integration

### PR DESCRIPTION
**Pull request by mitchellsimoens**
**Summary**

Adds the Gimbal integration to the readme

### CODE REVIEW

**Comment by mitchellsimoens** [comment]
I signed it!

**Comment by mitchellsimoens** [comment]
@patrickhulce ah ok, misread the subsection for integration (skipped over the "services" word tbh). Moved it to related projects.

File: readme.md [hunk 1]
```diff
@@ -304,6 +304,7 @@ This section details services that have integrated Lighthouse data. If you're wo
 ## Related Projects
 Other awesome open source projects that use Lighthouse.
 
+* **[Gimbal](https://labs.moduscreate.com/gimbal-web-performance-audit-budgeting)** - An [open source (MIT licensed)](https://github.com/ModusCreateOrg/gimbal) tool to meaure and analyze aspects of a web application and check them against thresholds. Users can produce Lighthouse reports along with checking built file sizes and unused source. Gimbal can also report these reports to a GitHub pull request.
```
Comment (exterkamp) [inline readme.md]:
```suggestion
* **[Gimbal](https://labs.moduscreate.com/gimbal-web-performance-audit-budgeting)** - An [open source (MIT licensed)](https://github.com/ModusCreateOrg/gimbal) tool used to measure, analyze, and budget aspects of a web application. Gimbal also integrates reports with GitHub pull requests.
```

**Comment by mitchellsimoens** [comment]
@exterkamp thanks for the suggestions! Looks like googlebot is complaining on cla signed now?

**Comment by patrickhulce** [comment]
> Looks like googlebot is complaining on cla signed now?

It tends to get confused when non-Googler commits get mixed with Googler commits for some reason. No action needed on your part @mitchellsimoens the Googlebot already knew you signed it!

**Pull request merged**